tests/phpunit/tests/formatting/ExcerptRemoveBlocks.php

```diff
@@ -22,7 +22,7 @@ class Tests_Formatting_ExcerptRemoveBlocks extends WP_UnitTestCase {
 	<!-- wp:column -->
 	<div class="wp-block-column">
 		<!-- wp:archives {"displayAsDropdown":false,"showPostCounts":false} /-->
-		
+
```

Comment:
@pbearne this change and the one on line 43 are impacting the failing test. Plus it's not needed as part of covers. Reverting the change here and on line 43 should resolve the failing test.